src/api/config/feature.yml

```diff
@@ -0,0 +1,2 @@
+features:
+  image_templates: true
```

Comment:
When a feature key is not defined in the file the gem thinks that this is disabled